functions/Test-DbaLastBackup.ps1

```diff
@@ -259,7 +268,37 @@ Restores data and log files to alternative locations and only restores databases
 					}
 					
 					$lastbackup = Get-DbaBackupHistory -SqlServer $sourceserver -Databases $dbname -LastFull -IgnoreCopyOnly:$ignorecopyonly
-										
+
+					if($lastbackup[0].Path.StartsWith('\\') -eq $false -and $CopyDestination -eq $true) 
+					{
+						Write-Message -Level Verbose -Message "Copying backup to destination server"
+						if($destserver.BackupDirectory.StartsWith('\\') -eq $false)
+						{
+							Write-Message -Level Verbose -Message "Destination backup directory is not UNC and source does not match destination."
+							$fileexists = "Skipped"
+							$restoreresult = "Destination backup not on shared location."
+							$dbccresult = "Skipped"
+						}
+						elseif ((Test-SqlPath -SqlServer $destserver -path $destserver.BackupDirectory) -eq $true) 
+						{
+							if((Test-SqlPath -SqlServer $destserver -path ("{0}\{1}" -f $destserver.BackupDirectory,$Prefix)) -eq $false)
```

Comment:
We have an internal command called Join-AdminUnc that will make your life easier. In this case, it'd be

`$copydirectory = Join-AdminUnc -servername $destserver.netname -filepath $lastbackup.Path`

Makes me realize that Join-AdminUnc function could actually use some fancying up -- detecting if its SMO or string and getting the netname if SMO but for now, that is the functionality.